lib/routes/gov/mem/zfxxgkpt.ts

```diff
@@ -0,0 +1,100 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/mem/gk/zfxxgkpt/fdzdgknr',
+    categories: ['government'],
+    example: '/gov/mem/gk/zfxxgkpt/fdzdgknr',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.mem.gov.cn/gk/zfxxgkpt/fdzdgknr'],
+            target: '/mem/gk/zfxxgkpt/fdzdgknr',
+        },
+    ],
+    name: '应急管理部法定主动公开内容',
+    maintainers: ['skeaven'],
+    handler,
+    description: `无参数`,
+};
+
+async function handler(ctx) {
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 30;
+
+    const rootUrl = 'https://www.mem.gov.cn';
+    const currentUrl = new URL(`gk/zfxxgkpt/fdzdgknr`, rootUrl).href;
```

Comment:
```suggestion
    const currentUrl = new URL(`gk/zfxxgkpt/fdzdgknr/`, rootUrl).href;
```
Use the [redirected](https://github.com/DIYgod/RSSHub/actions/runs/17324468979/job/49185024935#step:13:19) URL instead